config/default.json

```diff
@@ -197,16 +197,18 @@
           "'self'",
```

Comment:
Suspect the changes in this file are not needed, as the config is via Helmet in the `application.ts`, but changing here anyway just in case and for consistency.